### ISSUE TITLE
Improve packaging & docs for non-developers

### ISSUE THREAD

**Issue by ttmc** [oceanprotocol/aquarius#93]
Aitor asked me to make an issue summarizing some of the things we could do to make Aquarius more usable for non-developers (Aquarius "operators").

## Must-Have for Trilobite

- [x] 1 - `pip install ocean-aquarius` (from PyPI) should install all its requirements and not packages only needed by Aquarius developers. (Aquarius developers can have their own installation process.) Don't worry about BigchainDB as a requirement, for now. This task should be resolved once:
  - PR #96 is merged _and_
  - a new version of `ocean-aquarius` is released on PyPI.
- [x] 2 - Minimal docs for Aquarius _operators_ about how to install, configure and run Aquarius _somehow_ (probably using `apt` and `pip`). This includes running MongoDB or Elasticsearch in a more production-grade way. This will be done, for now, once PR #105 is merged.
- [x] 3 - Answers to questions about Keeper nodes, e.g. "As an Aquarius node operator, what Keeper nodes can I connect it to? Is a local one the only option? Are there public ones I can connect to for testing?"

## Nice-to-Have for Trilobite

I moved all of these to new issues of their own.
